themes/default/content/learn/pulumi-fundamentals/create-docker-images/index.md

```diff
@@ -78,6 +78,20 @@ host machine.
 
 ## Build your Docker Image with Pulumi
 
+{{% choosable language typescript %}}
```

Comment:
Random question: When I switch back and forth on the choosable, the text column width also changes. It looks like the code box doesn't have a horizontal scroll. Think that's worth fixing?

Reply:
Sure!
A | B
-- | --
![Screen Shot 2022-02-09 at 4 18 04 PM](https://user-images.githubusercontent.com/1538692/153299769-e1631ec2-d432-448c-b04d-353227b98e92.png) | ![Screen Shot 2022-02-09 at 4 18 10 PM](https://user-images.githubusercontent.com/1538692/153299771-71e1fd60-181b-4fd4-9d44-a74faa66a68a.png)
![Screen Shot 2022-02-09 at 4 18 18 PM](https://user-images.githubusercontent.com/1538692/153299772-04df0bfd-f59d-4a8a-93c2-89220716115d.png) | ![Screen Shot 2022-02-09 at 4 18 25 PM](https://user-images.githubusercontent.com/1538692/153299775-35d17474-ac3d-440b-954b-d7a55759c1ff.png)

Reply:
Ah, okay. It looks like the code is extending the width of the column as much as it can and _then_ side scrolling. I'll make an issue for this on the theme repo, which is probably the right place for a fix for this. (We probably just need to apply an explicit full width to these code blocks.)